containers/nginx-storage/install-dependencies.sh.j2

```diff
@@ -40,4 +40,7 @@ rm -rf /opt/lua-resty-http
 # Lua libs focusing on input data handling, functional programming and OS path management.
 /usr/local/openresty/luajit/bin/luarocks install penlight
 
+mkdir /srv/storage/
+chown -R nobody:nobody /srv/storage/
+
```

Comment:
This this change related to this pull request?

It should at least be in a separate commit.

Reply:
Yes, those settings allows to upload file to docker image. Without this, upload to nginx-storage locally not working.

Reply:
Done.